paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -175,6 +175,7 @@ class PaparazziPlugin : Plugin<Project> {
       val testTaskProvider = project.tasks.named("test$testVariantSlug", Test::class.java) { test ->
         test.systemProperties["paparazzi.test.resources"] =
           writeResourcesTask.flatMap { it.paparazziResources.asFile }.get().path
+        test.systemProperties["paparazzi.project.dir"] = project.projectDir.path
```

Comment:
can we follow the pattern for buildDirectory and do something like this?

```suggestion
        test.systemProperties["paparazzi.project.dir"] = projectDirectory.toString()
```

This means extracting `val projectDirectory = project.layout.projectDirectory` from `snapshotOutputDir` above.